cmd/fossa/main.go

```diff
@@ -244,6 +252,10 @@ func defaultCmd(c *cli.Context) {
 		mainLogger.Fatalf("Could not load configuration: %s", err.Error())
 	}
 
+	if ok, err := checkUpdate(); ok && err == nil {
+		mainLogger.Infof("An update is available for this CLI; run `fossa update` to get the latest version.")
```

Comment:
This won't actually log anything for most users. See logging levels from `go-logging`:
```go
const (
	CRITICAL Level = iota
	ERROR
	WARNING
	NOTICE
	INFO
	DEBUG
)
```
We currently only log `WARNING` and above by default (see `initialize` in `config.go`). I'm alright with changing the default logging level to `INFO` and above, although I'd actually consider this a `NOTICE`.